docs/components/button/index.html

```diff
@@ -217,5 +214,53 @@ <h2 id="button-group">Button Group</h2>
         {% endcode %}
       </footer>
     </div>
+    <p class="hxSubdued hxSubBody">
+      <hx-icon type="exclamation-triangle"></hx-icon>
+      Size and Weight classes are applied to the <em>entire</em> bar, not to individual buttons.
+    </p>
+  </section>
+
+  {# BUTTON SET #}
```

Comment:
New `Button Set` demo (SURF-1705).

**TODO:** Documentation `Button Set` usage in dev docs. (SURF-1986)